From 450d5dee6cc4f35a7475af507b90ba67327ad724 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 26 Apr 2016 17:48:18 +0200
Subject: [PATCH] qdeclarativetextinput: update to match QWidgetLineControl API
 change

* setCursorBlinkPeriod was replaced with setBlinkingCursorEnabled &
  updateCursorBlinking in qtbase commit:
  commit 1b5bc9723c0b23ca5197097c3087df6bbe024a2a
  Author: Richard Moe Gustavsen <richard.gustavsen@theqtcompany.com>
  Date:   Tue Apr 19 10:15:54 2016 +0200
  Subject: QWidgetLineControl: respect run-time changes to cursorFlashTime

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/declarative/graphicsitems/qdeclarativetextinput.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/declarative/graphicsitems/qdeclarativetextinput.cpp b/src/declarative/graphicsitems/qdeclarativetextinput.cpp
index 3ff75fd..e39d1af 100644
--- a/src/declarative/graphicsitems/qdeclarativetextinput.cpp
+++ b/src/declarative/graphicsitems/qdeclarativetextinput.cpp
@@ -514,7 +514,7 @@ void QDeclarativeTextInput::setCursorVisible(bool on)
     if (d->cursorVisible == on)
         return;
     d->cursorVisible = on;
-    d->control->setCursorBlinkPeriod(on?QApplication::cursorFlashTime():0);
+    d->control->setBlinkingCursorEnabled(on);
     QRect r = d->control->cursorRect();
     if (d->control->inputMask().isEmpty())
         updateRect(r);
